ci/Jenkinsfile.groovy

```diff
@@ -62,8 +62,11 @@ DOCKER_IMAGE_X86_64_MANYLINUX = "quay.io/pypa/manylinux2010_x86_64"
 // Note: global variables must be declared without `def`
 //       see https://stackoverflow.com/questions/6305910
 
-// Needs invocation of larger tests
-needsLargerTest = false
+// Larger tests are invoked if forced or if isModified() routine
+// returns `true`, and unless the larger tests are explicitely
+// disabled. Here we only define the default value for `needsLargerTests`
+// adjusting it when we check for the actual code changes.
+needsLargerTests = params.FORCE_LARGER_TESTS || params.FORCE_ALL_TESTS
```

Comment:
Can we please call this variable `doFreadTests`, similar to all other variables in lines 71-75.
Likewise, the param name can be `FORCE_FREAD_TESTS`. This would reduce the confusion, I think.

Reply:
Yeah, I already renamed it. Just didn't push as waiting for the larger tests to finish.

Reply:
As for the `FORCE_FREAD_TESTS`, I'm not sure. Can we have larger tests for other datatable functions? For example larger sort, join or groupby tests that may be optional and controlled with the same flag? 

In any case I guess it's better to keep the word `LARGER` in this variable, because it doesn't control the other fread tests. Something like `FORCE_LARGER_FREAD_TESTS`.

Reply:
I mean, "large tests" doesn't really reflect the purpose of the variable. It really only controls extra fread tests. So we could call it `doLargeFreadTests` or `doExtraFreadTests` or `doFreadTests`, whereas "do large tests" could be used to execute tests with >2B rows frames or >2GB string columns or similar.

Reply:
Right now — yes. I thought there was an idea for that variable to control some other larger tests? Or it got its name just by chance?

Reply:
If we need to control the execution of really large tests, that should probably be a separate variable.

Reply:
Make sense. I will rename it once the larger tests finish.

Reply:
Btw, I guess the variable name is coming from the datatable environment variable `DT_LARGE_TESTS_ROOT`. This one has no indication it is only applicable to fread tests.